### DISCUSSION TITLE
Adding IOS Widget

### DISCUSSION THREAD

**Discussion by 7urkm3n** [OutlineFoundation/outline-apps#1304] [Ideas]
Hi Outline community,

Who ever has an experience in Cordova/IOS, could you able to add a Widget for IOS/Android (especially IOS). 
Its annoying every time going to Outline app and connect to the server and stop it later, since some of us don't use it full-time. 

Sample widget pic was taken from "CyberGhostVpn"

![IMG_0588](https://user-images.githubusercontent.com/2600920/167659701-a542dd83-efb3-49a7-8d46-6a02c5d058b6.jpg)

I am ready to guide and help, just bad at mobile development :) 

Thanks,
Regards

**Comment by bemasc** [comment]
See #1297 for a related request on Android

**Comment by 7urkm3n** [comment]
Wow, since 2022 nobody has tried to implement the feature ?
